.github/workflows/build.yml

```diff
@@ -1,13 +1,17 @@
 name: Build Workflow
 
 on:
+  workflow_dispatch:
```

Comment:
Why do we need manual trigger?

Reply:
It is nice to have to be able to retrigger builds on our main branch

Reply:
If retrigger a run on existing change, can't we rerun-all-jobs in each workflow?

Reply:
This repository changes relatively infrequently. When we discover issues with the pushed artifact, we often do not have the logs of the built artifact because they have expired. Having a manual way to retrigger these debian package builds allows us to more easily trace issues with the build as well as ensure that we can build this software after it's dependencies change (carma-builds) without having to make a superfluous change to trigger a build. We can not retrigger builds if nothing in the repo has changed without this manual trigger.